layers/gtp.go

```diff
@@ -176,6 +176,9 @@ func (g *GTPv1U) NextLayerType() gopacket.LayerType {
 	if len(g.LayerPayload()) == 0 {
 		return gopacket.LayerTypeZero
 	}
+	if g.MessageType != 255 {
```

Comment:
where does this definition come from? I'm looking at 3GPP TS 29.060 section 7.1 and it doesn't align with this condition.

Reply:
The document you are referring to defines T-PDU (tunneled traffic such as IPv4 or IPv6) and G-PDU in section 3.

In 7.1 it says that G-PDU is message type 255.